<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>760</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>680</width>
    <height>760</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>680</width>
    <height>760</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>410</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>ЗАПУСК</string>
    </property>
   </widget>
   <widget class="QLabel" name="kklabel">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>130</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>КИС-2</string>
    </property>
   </widget>
   <widget class="QLabel" name="saplabel">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>130</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>ПРДЦ</string>
    </property>
   </widget>
   <widget class="QLabel" name="tmblabel">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>130</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>КИС-1</string>
    </property>
   </widget>
   <widget class="QComboBox" name="tmb_comboBox">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>160</y>
      <width>73</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="kk_comboBox">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>160</y>
      <width>73</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="sap_comboBox">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>160</y>
      <width>73</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="aztmb_lineEdit">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>220</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="azkk_lineEdit">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>220</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="azsap_lineEdit">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>220</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="azimut_label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>220</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Азимут</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ugolmtmb_lineEdit">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>290</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="ugolm_label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>290</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Угол места</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ugolmkk_lineEdit">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>290</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="object_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Объект наблюдения</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ugolmsap_lineEdit">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>290</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="list_comboBox">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>50</y>
      <width>411</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="com_label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>160</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>COM-PORT</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loadButton">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>50</y>
      <width>51</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Load</string>
    </property>
   </widget>
   <widget class="QPushButton" name="closeButton">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>410</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>ЗАКРЫТЬ ПОРТЫ</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="Date_Edit">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>390</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:16pt; font-weight:600; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:7.8pt; font-weight:400;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="log_textBrowser">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>540</y>
      <width>661</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>510</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Logs write...</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>370</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Дата</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>450</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Московское время</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcd_TimeEdit">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>470</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QRadioButton" name="autoRadioButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>370</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Автоматическое</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="handRadioButton">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>370</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Программное</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>350</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Сопровождение КА</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
